src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -8,9 +8,53 @@
 #include <time.h>
 #include <float.h>
 #include <limits.h>
+#include <ctype.h>
 
 #include "lfortran_intrinsics.h"
+#include "../config.h"
```

Comment:
CI fails because of this.

Reply:
I think we can do `#include <libasr/config.h>` and then pass the properly include path to `libasr` root directory using cmake.